streams/jvm/src/test/scala/scalaz/zio/stream/StreamSpec.scala

```diff
@@ -529,4 +530,11 @@ class StreamSpec(implicit ee: org.specs2.concurrent.ExecutionEnv)
         }
     ) must_== Failure(Cause.Fail(e))
   }
+
+  private def bufferStream = {
```

Comment:
A property check could be used here to generate a wider range of test-cases.

Reply:
Fixed